packages/commonwealth/server/controllers/server_groups_methods/delete_group.ts

```diff
@@ -0,0 +1,21 @@
+import { ChainInstance } from 'server/models/chain';
+import { ServerChainsController } from '../server_chains_controller';
+import { AddressInstance } from '../../models/address';
+import { UserInstance } from '../../models/user';
+
+export type DeleteGroupOptions = {
```

Comment:
Are all of these fields mandatory?

Reply:
Yes, for the admin check and potentially ban check if we want to do that.

Reply:
Well, in theory an admin can't be banned, so we shouldn't need to do a ban check. But, got it.